templates/home.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+    
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <link href="{{url_for('static',path='/css/main.css')}}" rel="stylesheet">
+        <title>Hello</title>
+    </head>
+    <body>
+        
+        <h1>HELLO WORLD.</h1>
```

Comment:
nice try using some js to display the response of this: https://github.com/stephenjk41/fastapi-sample/blob/main/fast-api-service/routers/sample_route.py#L5 in the html dom